warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/handler/edge/EdgeKeyVersioningCache.java

```diff
@@ -148,13 +148,11 @@ public void updateCache(FileSystem fs) throws AccumuloSecurityException, Accumul
         }
         
         // now attempt to write them out
-        try {
-            PrintStream out = new PrintStream(new BufferedOutputStream(fs.create(tmpVersionFile)));
+        try (PrintStream out = new PrintStream(new BufferedOutputStream(fs.create(tmpVersionFile)))) {
             
             for (Map.Entry<Integer,String> pair : versionDates.entrySet()) {
                 out.println(pair.getKey() + "\t" + pair.getValue());
             }
-            out.close();
             
```

Comment:
I believe the file needs to be close here before we attempt to rename it below.

Reply:
Fixed in next commit by ensureing a local twr